### DISCUSSION TITLE
User interface review

### DISCUSSION THREAD

**Discussion by krishnasaurabh** [ramyasaimullapudi/WolfTrackPlus#18] [Ideas]
How can we make the user interface better, so it makes the user feel the application is seamless and wants to keep coming back.

**Comment by arjunptm** [comment]
The user interface is good, but I think it can benefit from having scroll bars.

**Comment by krishnasaurabh** [comment]
![image](https://user-images.githubusercontent.com/12223198/144288612-5d06150c-20a3-4076-8fcb-981dd0f548ab.png)
mock design for the scroll bars
